dev-guide/ci/events-files.md

```diff
@@ -0,0 +1,66 @@
+    file in that same location
+  * [Example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/gather-must-gather/artifacts/event-filter.html)
+  * Tip: wait several seconds for the html page to finish rendering so you can use the text entry boxes to search
+
+* events.json (same as `oc_cmds/events` mentioned below except as json)
+  * generated via `oc get events --all-namespaces -o json” (see [source code](https://github.com/openshift/release/blob/f5017d5136a740a4186477b02bed70047ade200b/ci-operator/step-registry/gather/extra/gather-extra-commands.sh#L61))
+  * located in the prow job stored artifacts at `.../gather-extra/aftifacts/`
+  * [Example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/gather-extra/artifacts/events.json)
+
+ * events (same as `gather-extra/artifacts/events.json` except not json)
+   * generated via `oc get events --all-namespaces` (see [source code](https://github.com/openshift/release/blob/f5017d5136a740a4186477b02bed70047ade200b/ci-operator/step-registry/gather/extra/gather-extra-commands.sh#L62)).  This file has all the events present in the apiserver at the time it was generated.
+   * located in the prow job stored artifacts at `.../gahter-extra/artifacts/oc_cmds/events`
+   * [Example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/gather-extra/artifacts/oc_cmds/events)
+
+* resource-events_(date)-(number).zip
+  * located in the prow job stored artifacts at `.../<test_name>/openshift-e2e-test/artifacts/junit`
+  * There will be one file per invocation of `openshift-tests`
+  * Contains a watch of all events including "delete" related events
+  * These contain kube-apiserver events
+  * [Example1](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/openshift-e2e-test/artifacts/junit/resource-events_20220321-040307.zip), [Example2](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/openshift-e2e-test/artifacts/junit/resource-events_20220321-052002.zip)
+  * Here's a sample of the contents:
+
+```
+    ./default:
+    total 332
+    -rw-rw-r-- 1 dperique dperique 339013 Dec 31  1979 events.json  <-- why the ancient date?
```

Comment:
that's the Unix epoc, and is just showing the timestamp property was left unset.